metal3-dev/local-cbo.sh

```diff
@@ -14,10 +14,12 @@ if [ ! -d $cbo_path ]; then
     exit 1
 fi
 
+export IMAGES_JSON=$(realpath ${OCP_DIR}/cbo-images.json)
```

Comment:
This won't work when running the script from the metal3-dev dir, should we perhaps use `$SCRIPTDIR` instead?

Reply:
> This won't work when running the script from the metal3-dev dir, should we perhaps use `$SCRIPTDIR` instead?

Done, thanks.